src/test/java/teammates/test/cases/browsertests/FeedbackContributionQuestionUiTest.java

```diff
@@ -180,7 +180,7 @@ private void testCopyOptions() {
         feedbackEditPage.clickNewQuestionButton();
         feedbackEditPage.selectNewQuestionType("CONTRIB");
         feedbackEditPage.fillQuestionTextBoxForNewQuestion("q2, contribution qn");
-        assertEquals("Please select a visibility option",
```

Comment:
The default visibility for contrib question is `Shown anonymously to recipient and team members, visible to instructors`, which is what the code was doing. I removed it, but it was actually fine. The code just needed one line, setting the button text back to `Please select a visibility option`. Working on it.